### ISSUE TITLE
Entering an existing directory name for create remote directory gives The remote server returned an error: (550) File unavailable (e.g., file not found, no access).

### ISSUE THREAD

**Issue by saraswathidatar** [pdx-cs-lbryan/ftpclient#1]

**Comment by pdx-cs-lbryan** [comment]
I'm not sure if this is addressable in a good way. I've spent some time trying to make this more user friendly, and the solution I've come up with isn't very clean. The error is not wrong -- the correct protocol error is being detected and sent to the user. However, as noted, this error code covers a variety of possibilities. 

Ideally I'd be able to just use the response and get the status code, detect if its 550, and do some unique processing. However, with the library being used, a 550 error immediately sends a webexception. Now having to use the catch block, the try block variable is lost, which means the response code can't be used.

The solution I came up with was just to see if the error message matched the string as listed in the issue to make it a little more accurate to the circumstances.

This is addressed in the 'new-createdir-unittests branch'
